lib/encoder.js

```diff
@@ -782,7 +797,8 @@ function encode(imgData, qu) {
   return {
     data: data,
     width: imgData.width,
-    height: imgData.height
+    height: imgData.height,
+	comments: imgData.comments
```

Comment:
```suggestion
```

sorry I missed this, I don't think we should return here. is there a reason you need to?

Reply:
There is no specific reason. It was more to have the same output in both functions.